owtf/webapp/src/containers/Report/Table.tsx

```diff
@@ -244,9 +244,9 @@ export class DataTable extends React.Component<propsType ,stateType> {
                       {resourceList.map(
                         (resource, index) => 
                         <li key={index}>
-                          <Link to={resource[1]} target="__blank">
+                          <a href={resource[1]} target="__blank">
```

Comment:
`<Link>` is a React router component and is rendered into a `<a>` tag correctly if the Javascript code builds.

Reply:
![image](https://github.com/owtf/owtf/assets/32198580/45fceef5-f040-4fa2-bddd-9d1df495cbb8)
Hey! the current implementation of Link tag only allows internal navigation and does not allow navigation to external sites. Could you please correct me if I m wrong.
[](https://reactrouter.com/en/main/components/link)

Reply:
oh I didn't know this changed.